src/actions/store-actions.js

```diff
@@ -116,9 +117,16 @@ export function undefineVm({ connectionName, name, id, transientOnly }) {
     };
 }
 
-export function updateLibvirtState(state) {
+export function updateLibvirtService(state) {
     return {
-        type: UPDATE_LIBVIRT_STATE,
+        type: UPDATE_LIBVIRT_SERVICE,
+        state,
+    };
+}
+
+export function updateLibvirtSocket(state) {
```

Comment:
You don't use this anywhere at this point - just put into the store data which you plan to use in other places in the code.